src/core_modules/capture-core/components/WorkingLists/EventWorkingLists/epics/getEventListData.js

```diff
@@ -103,14 +104,25 @@ const getApiCategoriesQueryArgument = (categories: ?{ [id: string]: string}, cat
     if (!categories || !categoryCombinationId) {
         return null;
     }
+    const newUIDsSeparator = featureAvailable(FEATURES.newAocApiSeparator);
```

Comment:
Could we rename to `newUIDsSeparator` in [support.js](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core-utils/featuresSupport/support.js#L12) too?